src/widgets/about-course/ui/about-course-grid/about-course-grid.test.tsx

```diff
@@ -0,0 +1,55 @@
+import { render, screen } from '@testing-library/react';
+import { describe, expect, it } from 'vitest';
+import { AboutCourseGrid } from './about-course-grid';
+import { MOCKED_IMAGE_PATH } from '@/shared/__tests__/constants';
+
+const mockedData = [
+  {
+    id: 1,
+    title: 'Title 1',
+    info: 'Info 1',
+    icon: MOCKED_IMAGE_PATH,
+    alt: 'facts about the course - Title 1',
+  },
+  {
+    id: 2,
+    title: 'Title 2',
+    info: 'Info 2',
+    icon: MOCKED_IMAGE_PATH,
+    alt: 'facts about the course - Title 2',
+  },
+];
+
+describe('AboutCourseGrid component', () => {
+  let itemElements: HTMLElement[];
+
+  beforeEach(() => {
+    render(<AboutCourseGrid items={mockedData} />);
+    itemElements = screen.getAllByTestId('about-course-grid-item');
+  });
+
+  it('renders correct number of items', () => {
+    expect(itemElements).toHaveLength(mockedData.length);
+  });
+
+  it('renders correct content for each item', () => {
+    itemElements.forEach((item, index) => {
+      const titles = screen.getAllByTestId('subtitle');
+      const infoArray = screen.getAllByTestId('paragraph');
+
+      expect(item).toBeVisible();
+      expect(titles[index].textContent).toBe(mockedData[index].title);
+      expect(infoArray[index].textContent).toBe(mockedData[index].info);
+    });
+  });
```

Comment:
There is a better built in loop in `vitest` api

https://vitest.dev/api/#test-each

<img width="396" alt="Screenshot 2024-09-30 at 18 33 42" src="https://github.com/user-attachments/assets/4e73c40d-c2fc-4d41-9033-6b97155aaea2">

Reply:
Done